src/routes/safe/components/Transactions/helpers/EditableTxParameters.tsx

```diff
@@ -13,6 +13,7 @@ type Props = {
   parametersStatus?: ParametersStatus
   ethGasLimit?: TxParameters['ethGasLimit']
   ethGasPrice?: TxParameters['ethGasPrice']
+  ethMaxPrioFee?: TxParameters['ethMaxPrioFee']
```

Comment:
Instead of manually picking each key, could you not simply `Pick<TxParameters, 'x', 'y', 'z'>`?

Reply:
That could save some lines but in this case I think it is more readable the way it is. See how later we pass the params to `useTransactionParameters`

Reply:
Sure, it is a question of preference but reuse is at the heart of types.

Reply:
I mean, it's a definitely a question of preference but I would argue that it's safer when they are exactly the same. We can mark this as resolved though.